integration/src/test/resources/org/mule/test/config/correlation-id/static-generation.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<mule xmlns="http://www.mulesoft.org/schema/mule/core"
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xmlns:http="http://www.mulesoft.org/schema/mule/http"
+      xsi:schemaLocation="
+       http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd
+       http://www.mulesoft.org/schema/mule/http http://www.mulesoft.org/schema/mule/http/current/mule-http.xsd">
+
+    <configuration correlationIdGeneratorExpression="'doge'"/>
```

Comment:
this should fail on deployment

Reply:
add a validation for this. Ref: https://github.com/mulesoft/mule/tree/master/runtime-extension-model/src/main/java/org/mule/runtime/config/internal/validation

Reply:
Isn't it a valid expression?